### ISSUE TITLE
Data Generator produces empty values labelled as LOCATION

### ISSUE THREAD

**Issue by gabechu** [microsoft/presidio-research#22]
Here's a snippet of data I get from using the data generator, the problem is it includes invalid entities.

```
{
        "full_text": "The title refers to  Street in Cite Ezzitoun 2. It was on this street that many of the clubs where Metallica first played were situated. \"Battery is found in me\" shows that these early shows on Rue de Ouerdanine Street were important to them. Battery is where \"lunacy finds you\" and you \"smash through the boundaries.\"",
        "masked": null,
        "spans": [
            {
                "entity_type": "LOCATION",
                "entity_value": "",
                "start_position": 20,
                "end_position": 20
            },
            {
                "entity_type": "LOCATION",
                "entity_value": "Cite Ezzitoun 2",
                "start_position": 31,
                "end_position": 46
            },
...
```

For the very first span emitted from the generator, you would see that it is an empty LOCATION entity with start_position the same as end_position. This pattern has been repetitively showing up in my generated data.

This is the code I was using to generate the data, as there are some randomness, you may be able to reproduce the issue.
```
from presidio_evaluator.data_generator.main import generate

import datetime

EXAMPLES = 500
SPAN_TO_TAG = True  # Whether to create tokens + token labels (tags)
TEMPLATES_FILE = 'presidio_evaluator/data_generator/raw_data/templates.txt'
KEEP_ONLY_TAGGED = False
LOWER_CASE_RATIO = 0.1
IGNORE_TYPES = None

cur_time = datetime.date.today().strftime("%B_%d_%Y")
OUTPUT = "data/generated_size_{}_date_{}.json".format(EXAMPLES, cur_time)

fake_pii_csv = 'presidio_evaluator/data_generator/' \
               'raw_data/FakeNameGenerator.com_3000.csv'
utterances_file = TEMPLATES_FILE
dictionary_path = None

generate(fake_pii_csv=fake_pii_csv,
         utterances_file=utterances_file,
         dictionary_path=dictionary_path,
         output_file=OUTPUT,
         lower_case_ratio=LOWER_CASE_RATIO,
         num_of_examples=EXAMPLES,
         ignore_types=IGNORE_TYPES,
         keep_only_tagged=KEEP_ONLY_TAGGED,
         span_to_tag=SPAN_TO_TAG)
```

**Comment by omri374** [comment]
Hi @gabechu, thanks for adding your issue.
A few questions:
1. Are you using a downloaded dataset from FakeNameGenerator?
2. The data generator takes its location from one of these columns in the provided fake dataset: "LOCATION", "CITY", "STATE", "COUNTRY", "ADDRESS", "STREET". Could you please check if any of these have missing values?

**Comment by omri374** [comment]
Fixed in #43
